src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -1762,7 +1804,7 @@ let deleteSelectedExcelColumns (selectedColumns: seq<int>) (excelTable: Table) =
 /// <param name="newBB"></param>
 /// <param name="table"></param>
 /// <param name="context"></param>
-let addBuildingBlockAt (excelIndex: int) (newBB: CompositeColumn) (table: Table) (context: RequestContext)=
+let addBuildingBlockCellAt (excelIndex: int) (newBB: CompositeColumn) (table: Table) (context: RequestContext) =
```

Comment:
Wyh the rename? This adds a CompositeColumn right? therefore it adds a building block and not a cell

Reply:
I doubled he old method for testing and forgot to rename it afterwards

Reply:
fixed